python/ray/train/xgboost/v2.py

```diff
@@ -53,17 +51,16 @@ def train_fn_per_worker(config: dict):
                 "max_depth": 2,
             }
 
-            # 2. Do distributed data-parallel training with the `CommunicatorContext`.
+            # 2. Do distributed data-parallel training.
             # Ray Train sets up the necessary coordinator processes and
```

Comment:
Shall we mention `CommunicatorContext` here?

Reply:
I think there's no need to explain it in this example since the user shouldn't need to think about it. I could add it to the `XGBoostConfig` docstring since that's how users would customize the params passed into the `CommunicatorContext`.